changelog/@unreleased/pr-157.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Check log level enabled before evaluating parameters
```

Comment:
Can you expand the description a bit? For the change as implemented, would be something like:

```
Adds "Enabled" function to the LeveledLogger interface and updates logger implementations so that, if a logger is not enabled at a given level, the logger parameters are not evaluated.
```